src/js/components/DataTable/Body.js

```diff
@@ -44,6 +48,7 @@ const Row = memo(
       <StyledDataTableRow
         ref={rowRef}
         size={size}
+        tabIndex={onClickRow ? -1 : undefined}
```

Comment:
Why is this needed? I wasn't expecting us to be giving focus to the row.

Reply:
I played around with this and the tabIndex was needed in order to use the `onFocus` and `onBlur` properties but since we don't want to put focus on the row, I set it to `-1` to counteract that.